test/integration/kubernetes_deploy_test.rb

```diff
@@ -195,4 +195,18 @@ def test_wait_false_ignores_non_priority_resource_failures
     assert_equal 1, pods.size, "Unable to find web pod"
     assert_equal "Pending", pods.first.status.phase
   end
+
+  def test_extra_bindings_should_be_rendered
+    deploy_fixtures('extra-bindings', subset: ["web.yml.erb"], wait: false,
```

Comment:
If you want to make it faster, since it doesn't really matter what type of object you're templating into, you could make your new fixture a simple configmap instead, and check its data.